bindings/rust/s2n-tls-sys/Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "s2n-tls-sys"
 description = "A C99 implementation of the TLS/SSL protocols"
-version = "0.0.25"
+version = "0.0.26"
 authors = ["AWS s2n"]
 edition = "2021"
-rust-version = "1.60"
+rust-version = "1.63"
```

Comment:
Oh, nice catch :\ Does this not have any effect on things?

Reply:
It emits an error when someone tries to go build the crate on a lower version, which is a lot nicer than just getting some random compilation error.

Reply:
It would be nice if we could do something like https://github.com/aws/s2n-tls/commit/a107a34256ca9b12c7156ca023635b594a0b3984 for this then. Should be almost as easy. Something like 
`
grep "rust-version = \"$(cat ${{env.ROOT_PATH}}/rust-toolchain)\"" Cargo.toml
`

Reply:
Yes, definitely a fast followup item.